perf_bench_test.go

```diff
@@ -640,9 +645,13 @@ func calculateRPS(vmNum int) int {
 	return vmNum * baseRPS
 }
 
-func bindVMsToSocket() error {
+// bindVMs can bind VMs to cores.
+// If socket is set, it binds all VMs to the socket.
```

Comment:
what is "set"? set to a non-negative value? please be precise here and further in this comment

Reply:
Done